ci/taos/webapp/install-packages-nnstreamer.sh

```diff
@@ -42,16 +42,19 @@ echo -e "[DEBUG] Installing dependent packages by reading debian/control file."
 sudo mk-build-deps --install debian/control > /dev/null
 popd
 
-echo -e "########## for CI-system: Installing base packages for nnstreaemr app test "
+echo -e "########## for CI-system: Installing base packages for nnstreamer app test "
 sudo apt -y install rpmlint ctags sudo
 
 echo -e " Installing the 'meson=0.40' package requires 'xenial-backports'."
 sudo apt -y install ninja-build meson=0.40*
 
-echo -e "########## for CI-system: Installing python packages for nnstreaemr app test "
-sudo apt -y install python-gi python3-gi  
+echo -e "########## for CI-system: Installing python packages for nnstreamer app test "
+sudo apt -y install python-gi python3-gi
 sudo apt -y install python-gst-1.0 python3-gst-1.0
 sudo apt -y install python-gst-1.0-dbg python3-gst-1.0-dbg
 sudo apt -y install python-numpy python3-numpy
 
+echo -e "########## for CI-system: Installing pytorch packages for nnstreamer app test "
+sudo apt -y install pytorch
```

Comment:
Isn't this from our own PPA?
If so, you need related codes as well assuming that the user of this code is not aware of it. (add-ppa-repository commands that are ok with duplicated attempts)

Reply:
Yes, this is from our own PPA.
Our PPA is added on line 23 of this file.
Should I add again?

Reply:
Oh.no.. It's just me that missed that one.